client/go.mod

```diff
@@ -1,25 +1,57 @@
-module github.com/kubernetes-csi/external-snapshotter/client/v4
+module github.com/kubernetes-csi/external-snapshotter/client/v5
```

Comment:
My bad.  We actually need to update to client/v6.  Client/v5 was cut earlier but we didn't update this link.
https://github.com/kubernetes-csi/external-snapshotter/releases/tag/client%2Fv5.0.0

Can you submit another PR to update this to `module github.com/kubernetes-csi/external-snapshotter/client/v6`?